protocols/v2/subprotocols/common-messages/src/lib.rs

```diff
@@ -1,7 +1,15 @@
+#![warn(missing_docs)]
```

Comment:
Why do we need this?

Reply:
This will tell us if we missed adding docs for any public facing API